src/gtest/test_mnode_rpc.cpp

```diff
@@ -86,3 +90,32 @@ INSTANTIATE_TEST_SUITE_P(mnode_rpc_ani2psl, PTest_ani2psl_secret,
         }, true, true)
 	));
 
+TEST(ASCII85_Encode_Decode, ASCII85_Encode_Decode) {
```

Comment:
Test ASCII85_Encode_Decode is a perfect candidate for parameterized test.
You declare fixture class
class MyTestClass : public ::testing::TestWithParam<....>
and in your case you can use tuple<string, string>  as a parameter
then instead of TEST() - you write TEST_P()
get<0>(GetParam()) -  this is how you access parameters
So you need to write test only once for one comparison.
Then you declare parameterized test definition 
INSTANTIATE_TEST_SUITE_P(..., ....,
   ::testing::Values(
          make_tuple("encoded_string1", "decoded_string1"), 
          make_tuple("encoded_string2", "decoded_string2"), 
          .....

In tests like this it is much easier to add later on any additional special test cases.
It will be just one extra line in INSTANTIATE_TEST_SUITE_P.